ingest/importer/submission/entity_linker.py

```diff
@@ -0,0 +1,184 @@
+from ingest.importer.submission.entity import Entity
+from ingest.importer.submission.errors import LinkedEntityNotFound, InvalidLinkInSpreadsheet, MultipleProcessesFound
+
+
+class EntityLinker(object):
+
+    def __init__(self, template_manager):
+        self.template_manager = template_manager
+        self.process_id_ctr = 0
+
+    def process_links_from_spreadsheet(self, entity_map):
```

Comment:
The verb "process" in. the name of the function and the domain entity "process" are confusing.

Reply:
Won't touch this file for now.

Reply:
Renamed!